src/main/java/bio/terra/service/filedata/DrsService.java

```diff
@@ -789,6 +822,15 @@ private SnapshotSummaryModel getSnapshotSummary(UUID snapshotId) {
         snapshotId, snapshotService::retrieveSnapshotSummary);
   }
 
+  public String registerDrsAliases(List<DrsAliasModel> aliases, AuthenticatedUserRequest userReq) {
+    // Make sure the user is a steward by checking for list jobs action
+    samService.verifyAuthorization(
+        userReq, IamResourceType.DATAREPO, appConfig.getResourceId(), IamAction.LIST_JOBS);
```

Comment:
Is there a reason that we're checking this as part of the service method rather than in the controller method?  I would prefer to keep these checks in controllers as a convention to avoid worrying that we're not checking access correctly, but I'm open to seeing it differently.

I also think that we should be checking that the user is a `datarepo` `admin` rather than a `steward`.  IIRC, stewardship of a `datarepo` instance is an outdated notion and is a larger group than we intend it to be.  There may be other endpoints where we are making this mistake.

Here's a place where we restrict an endpoint to `datarepo` `admin`s:

https://github.com/DataBiosphere/jade-data-repo/blob/243b1c18f1e593fcef28ba53a85ff0bc2f5b7f48/src/main/java/bio/terra/app/controller/SearchApiController.java#L101-L104

Reply:
Looking at the [SAM config](https://github.com/broadinstitute/sam/blob/develop/src/main/resources/reference.conf#L1027), both admins and stewards have the `IamAction.LIST_JOBS` role, so I agree I think it makes sense to use the `IamAction.CONFIGURE` action instead!

Reply:
Yep, you're totally right.  I'm also going to drive by and make sure upgrades have to be run by admins too